src/list-data/request-action-mapping.ts

```diff
@@ -2,25 +2,31 @@ import { EntityTypes, NrRequestActionCodes } from '@/enums'
 import { RequestActionMappingI } from '@/interfaces'
 
 const EntityTypesBC = [
-  EntityTypes.FR,
-  EntityTypes.DBA,
+  EntityTypes.BC,
+  EntityTypes.C,
+  EntityTypes.CBEN,
+  EntityTypes.CC,
+  EntityTypes.CCC,
+  EntityTypes.CP,
   EntityTypes.CR,
-  EntityTypes.UL,
+  EntityTypes.CS,
+  EntityTypes.CUL,
+  EntityTypes.DBA,
+  EntityTypes.FI,
+  EntityTypes.FR,
   EntityTypes.GP,
-  EntityTypes.LP,
   EntityTypes.LL,
-  EntityTypes.CP,
-  EntityTypes.BC,
-  EntityTypes.CC,
-  EntityTypes.SO,
+  EntityTypes.LP,
   EntityTypes.PA,
-  EntityTypes.FI,
-  EntityTypes.PAR
+  EntityTypes.PAR,
+  EntityTypes.SO,
+  EntityTypes.UL
 ]
```

Comment:
I added 5 cont in types (since there is no cont in type for CP), and sorted this list.

C, CBEN, CCC, CUL, and CS.